docs/modules/serialization/pages/serialization.adoc

```diff
@@ -58,6 +59,18 @@ may incur costs on short-lived use cases
 
 * A factory and configuration must be implemented
 
+|All clients
+
+| xref:custom-serialization.adoc[Custom Serialization]
+| * Does not require class to implement an interface
+
+* Convenient and flexible
+
+* Can be based on `StreamSerializer` `ByteArraySerializer`
+|* Serialization interface must be implemented
```

Comment:
I'd suggest "Must implement serialization interface", however, there are multiple uses of this phrase on the page so either change all or none